packages/web/src/components/pages/Package.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Error } from 'components/layouts';
+import PackagePage from '../layouts/PackagePage/PackagePage';
+import { useParams } from 'react-router-dom';
+import { usePackageInfo } from '../../store/hooks/usePackageInfo';
+import { Helmet } from 'react-helmet';
+
+export function PackageView() {
+  const { '*': packageName } = useParams();
+  if (!packageName) {
+    return <Error host='' />; // TODO
+  }
+
+  const { packageInfo, isLoading, error } = usePackageInfo(packageName) ?? {};
+
+  if (packageName && !isLoading && (error || !packageInfo)) {
+    // TODO: 404 error page
+    return <Error host={packageName} message='Package is not found in the database' />;
```

Comment:
It would be great to have some kind of a `request package indexing button` here in order to estimate the feature potential.
CC @zardak

Reply:
This is great idea, could you please create an issue?

Reply:
Great idea, could you create an issue for it?